src/PaperG/FirehoundBlob/Facebook/Targeting/FacebookAudienceTargeting.php

```diff
@@ -59,6 +59,18 @@ public function getType()
         return $this->type;
     }
 
+    public function isValid()
+    {
+        if (empty($this->getType()) ||
+            empty($this->getIds()) ||
+            !is_array(($this->getIds()))
+        ) {
+            return false;
+        }
+
+        return true;
+    }
```

Comment:
Should be able to get rid of the if statement, and just return the check directly.